### PR TITLE
Add depdive readme and docstring

### PR DESCRIPTION
**Pull request by nasifimtiazohi**
Should be sufficient to publish version 0.1.0

### CODE REVIEW

**Comment by bmwill** [comment]
Lets add the following to the Cargo.toml
```
description = "<What this crate is about>"
repository = "https://github.com/diem/whackadep"
license = "Apache-2.0"
```

File: depdive/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Depdive
```
Comment (bmwill) [inline depdive/README.md:1]: You should look to adding badges to the top of the readme similar to what was done here: https://github.com/bmwill/diffy

**Comment by bmwill** [comment]
You may also want to add some documentation to the root of the crate library (lib.rs file) so that when documentation is generated you have something that isn't empty.

**Comment by nasifimtiazohi** [comment]
> You may also want to add some documentation to the root of the crate library (lib.rs file) so that when documentation is generated you have something that isn't empty.

you mean to expand on these lines https://github.com/nasifimtiazohi/whackadep/blob/583896ce7f349f30d74675e601fa90f282bbec51/depdive/src/lib.rs#L1-L4?

File: depdive/README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+[![Documentation (master)](https://img.shields.io/badge/docs-master-59f)](https://github.com/diem/whackadep/depdive)
```
Comment (bmwill) [inline depdive/README.md:3]: There is no documentation generated at that link. Assuming we publish, you should provide links to the crate documentation on docs.rs (and a link to crates.io) using links that will exist once we publish.

File: depdive/Cargo.toml [hunk 3]
```diff
@@ -1,7 +1,16 @@
 [package]
 name = "depdive"
 version = "0.1.0"
-authors = ["Brandon Williams <bmwill@fb.com>"]
+description = "Rust dependency analysis tool"
+repository = "https://github.com/diem/whackadep"
+homepage = "https://github.com/diem/whackadep/diem"
```
[the remark this answers is not in the document]
Comment (nasifimtiazohi) [reply depdive/Cargo.toml]: oops. thanks for catching that.

**Review by bmwill — DISMISSED** [review]

**Pull request merged**